java/arcs/sdk/android/storage/service/testutil/TestConnectionFactory.kt

```diff
@@ -66,10 +88,9 @@ class TestBindingDelegate(private val context: Context) : StorageServiceBindingD
   override fun unbindStorageService(conn: ServiceConnection) {
     val intent = bindings.remove(conn)
     serviceController.get().onUnbind(intent)
-    if (bindings.isEmpty()) {
-      serviceController.destroy()
-    }
```

Comment:
To address the issues with calling create/destroy off of the main thread causing Robolectric to crash, I've just removed this call to `destroy` completely. No tests seem to depend on it.